src/ucm/util/reloc.c

```diff
@@ -91,6 +91,19 @@ ucm_reloc_get_entry(ElfW(Addr) base, const ElfW(Phdr) *dphdr, ElfW(Sxword) tag)
     return 0;
 }
 
+static void *
+ucm_reloc_get_pointer(ElfW(Addr) base, const ElfW(Phdr) *dphdr, ElfW(Sxword) tag)
+{
+    uintptr_t entry;
```

Comment:
init entry inline, and add a space line after it

Reply:
Done!